scripts/src/main/resources/scripts/command/java

```diff
@@ -55,6 +55,19 @@ function doMigrate() {
   exit ${?}
 }
 
+function doDemoSetup() {
+  local mts_branch="develop"
+  local mts_repo="my-thai-star-${mts_branch}"
+  local mvn_dir="${DEVON_IDE_HOME}/workspaces/main/${mts_repo}/java/mtsj/"
+  doDownload "https://github.com/devonfw/my-thai-star/archive/${mts_branch}.zip"
+  doExtract "${DEVON_HOME_DIR}/Downloads/develop.zip"
+  cp -r "${DEVON_IDE_HOME}/updates/extracted/${mts_repo}" "${DEVON_IDE_HOME}/workspaces/main/${mts_repo}"
+  rm -rf "${DEVON_IDE_HOME}/updates/extracted/${mts_repo}"
+  pushd "${mvn_dir}" > /dev/null
```

Comment:
This `pushd` / `popd` encapsulation wonders me a bit here: Is it important, that the `doDevonCommand` is executed inside this special directory - even when the `eclipse import mvn` itself does not need it?

Is it, so only the mtsj - and nothing else from MTS - is imported? While reading this code, I more and more wonder, if the PATH parameter should not be instead two pathless arguments specifying the following

1. devonfw workspace name (relative)
2. sub-path inside project

Just some thoughts for now. I will need to make my own tests for this - for the moment I do not get the exact behavior from the code review only, Sorry.